rkt/pods.go

```diff
@@ -891,19 +890,51 @@ func (p *pod) getAppsHashes() ([]types.Hash, error) {
 	return hashes, nil
 }
 
+type AppsImageManifests map[types.ACName]schema.ImageManifest
+
+// getAppsImageManifests returns a map of ImageManifests keyed to the
+// corresponding App name.
+func (p *pod) getAppsImageManifests() (AppsImageManifests, error) {
+	apps, err := p.getApps()
+	if err != nil {
+		return nil, err
+	}
+
+	aim := make(AppsImageManifests)
+	for _, a := range apps {
+		imb, err := ioutil.ReadFile(common.AppInfoImageManifestPath(p.path(), a.Name))
+		if err != nil {
+			return nil, err
+		}
+
+		im := schema.ImageManifest{}
+		if err := im.UnmarshalJSON(imb); err != nil {
+			return nil, err
+		}
+	}
+
+	return aim, nil
```

Comment:
> @blixtra No worries. But I wonder why the tests haven't catch this.

Semephore did, but for some reason, it doesn't  show up as a big green check (red x in this case), but as a small dot beside the commit. Just buggy integration I assume.

I've fixed it now and it's all green. thx again.